internal/services/oracle/registration.go

```diff
@@ -15,6 +15,8 @@ func (r Registration) DataSources() []sdk.DataSource {
 	return []sdk.DataSource{
 		AdbsCharSetsDataSource{},
 		AdbsNCharSetsDataSource{},
+		AutonomousDatabaseCloneFromBackupDataSource{},
```

Comment:
Could you alpha sort the registrations?
```suggestion
		AutonomousDatabaseBackupDataSource{},
		AutonomousDatabaseBackupsDataSource{},
		AutonomousDatabaseCloneFromBackupDataSource{},
```